store/sqlstore/store.go

```diff
@@ -163,6 +165,30 @@ type ColumnInfo struct {
 	CharMaximumLength int
 }
 
+type jsonArray []string
```

Comment:
Let me know if you feel this should be moved to a new file. I think this file has accumulated a lot of cruft that can be moved to an adapters.go file or sth.

Reply:
Yeah that would be a great idea. I'd also add a small unit test for `Value()`, just for correctness sake.